include/rapidcheck/detail/Unicode.hpp

```diff
@@ -0,0 +1,96 @@
+#pragma once
+
+namespace rc {
+namespace detail {
+
+template<typename T, typename RandomType>
+T generateCodePoint(rc::detail::BitStream<RandomType>& stream)
+{
```

Comment:
RapidCheck coding guidelines require braces on the same line. There is a `.clang-format` file so your best bet is to run it on the whole thing.